src/components/day-night-theme/DayNightTheme.css

```diff
@@ -0,0 +1,59 @@
+.toggle {
+	display: flex;
+	align-items: center;
+	justify-content: center;
+}
+
+.toggle-text {
+	font-size: 18px;
+	color: var(--toggle-text);
+	margin: 0 10px;
+	display: flex;
+	align-items: center;
+	justify-content: center;
+}
+
+.toggle-button {
+	position: relative;
+	height: 24px;
+	width: 50px;
+	background-color: #5a537b;
+	border-radius: 50px;
+	padding: 3px;
+	cursor: pointer;
+	transition: background-color 250ms ease-in-out;
+}
+
+.toggle-button:hover {
+	background-color: #71699b;
+}
+
+.inner-circle {
+	position: absolute;
+	top: 2px;
+	left: 2px;
+	width: 20px;
+	height: 20px;
+	border-radius: 50%;
+	background-color: white;
+	transition: transform 250ms ease-in-out,
+		background-color 250ms ease-in-out;
+}
+
+.toggle-button.toggled .inner-circle {
+	transform: translateX(26px);
+	background-color: #333;
+}
+
+.unselected {
+	opacity: 0.5;
+}
+body {
```

Comment:
Nitpick: I think these should go in `App.css` so they are easier to find if someone is looking for the default colors.